cmd/tapcli/universe.go

```diff
@@ -581,6 +585,59 @@ func universeProofInsert(ctx *cli.Context) error {
 	return nil
 }
 
+var universeProofPushCommand = cli.Command{
+	Name:      "push",
+	ShortName: "p",
+	Usage:     "push a proof to a remote Universe",
+	Description: `
+	Push a proof present in the local Universe to a remote Universe.
+	`,
+	Flags:  append(universeProofArgs, universeServerArgs...),
```

Comment:
The `--server_id` flag has the text `the ID of the universe server to delete` which is a bit weird in this context. Might need to re-define it here.